src/font/lv_font_fmt_txt.h

```diff
@@ -156,6 +157,31 @@ typedef struct {
     uint32_t last_glyph_id;
 } lv_font_fmt_txt_glyph_cache_t;
 
+#if LV_USE_FONT_DYNAMIC_LOAD
+typedef struct {
+    /* Pointer to the font header structure. */
+    void * header;
```

Comment:
The header is font specific? Is not possible to have a standard type for it?

Reply:
`font_header_bin_t` is currently defined in `lv_font_loader.c`, and it's used here just to calculate nbits:
```
int nbits = header->advance_width_bits + 2 * header->xy_bits + 2 * header->wh_bits;
```
Would it be better to store `nbits` directly to avoid recalculating it multiple times, what are your thoughts?